Task2 2023/PhotoEnhancer/PhotoEnhancer/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Security.Cryptography;
+
+namespace PhotoEnhancer
+{
+    internal static class Program
+    {
+        /// <summary>
+        /// Главная точка входа для приложения.
+        /// </summary>
+        [STAThread]
+        static void Main()
+        {
+            Application.EnableVisualStyles();
+            Application.SetCompatibleTextRenderingDefault(false);
+
+            var mainForm = new MainForm();
+
+            mainForm.AddFilter(new PixelFilter<LighteningParameters>(
+                "Осветление/затемнение",
+                (pixel, parameters) => pixel * parameters.Coefficient));
+
+            Random rnd = new Random();
+            mainForm.AddFilter(new PixelFilter<NoiseParameters>(
+                "Уровень шума",
```

Comment:
Название для фильтра неудачное. Это название подходит для описания параметра. Лучше просто
```suggestion
                "Шум",
```